src/test/test-fs-util.c

```diff
@@ -864,14 +864,16 @@ static void test_path_is_encrypted_one(const char *p, int expect) {
 }
 
 static void test_path_is_encrypted(void) {
-        log_info("/* %s */", __func__);
+        int booted = sd_booted();
+
+        log_info("/* %s (sd_booted=%d)*/", __func__, booted);
 
         test_path_is_encrypted_one("/home", -1);
         test_path_is_encrypted_one("/var", -1);
         test_path_is_encrypted_one("/", -1);
-        test_path_is_encrypted_one("/proc", false);
-        test_path_is_encrypted_one("/sys", false);
-        test_path_is_encrypted_one("/dev", false);
+        test_path_is_encrypted_one("/proc", booted > 0 ? false : -1);
+        test_path_is_encrypted_one("/sys", booted > 0 ? false : -1);
```

Comment:
uh, if these two aren't the real things, then we are fucked... I wouldn't attempt to support that. Much of our API uses files in /sys and /proc, and they cannot be replaced by regular persistent file systems unlike /dev. I#d drop these two changes

Reply:
Ack.